src/features/Content/MoviePeople/Tile/index.js

```diff
@@ -19,12 +19,10 @@ const Tile = ({ movieBrowser, credits }) => {
     <>
       {creditsDisplay.map(({ id, name, profile_path, character, job }) => (
         <StyledLinkPeople key={id} to={`/people/${id}`}>
-          <ImageContainer  width="176px" height="264px" mobileWidth="120px" mobileHeight="178px">
-            {profile_path ?
-              <ImagePeople src={img_base_url + profile_path} alt="" />
-              : <ProfileIcon />
-            }
-          </ImageContainer>
+          {profile_path
+            ? <ImagePeople src={img_base_url + profile_path} alt="" />
+            : <ImageContainer width="176px" height="264px" mobileWidth="120px" mobileHeight="178px"> <ProfileIcon /> </ImageContainer>
+          }
           <Description>
             <Title align="center"> {name} </Title>
             {(character || job) ?
```

Comment:
Tak mi się wydaje, że tutaj dodać trzeba deklarację, że jeśli nie ma zdjęcia to wstawiamy zaślepkę?